stmhal/boards/ESPRUINO_PICO/mpconfigboard.h

```diff
@@ -2,6 +2,12 @@
 #define MICROPY_HW_MCU_NAME         "STM32F401CD"
 #define MICROPY_PY_SYS_PLATFORM     "pyboard"
 
+#undef  MICROPY_PY_USOCKET
```

Comment:
These undef's won't be necessary if mpconfigboard.h is included first in mpconfigport.h (see other comment).